Email Account & Probation Policy - Clerical updates (#476)
thewca/wca-documents: documents/policies/internal/Email Account.md

# WCA Email Account Policy

### Version 1.4 {.version}

## Purpose

> The purpose of this policy is to outline the guidelines and requirements of individual user accounts and group accounts on the worldcubeassociation.org domain. This policy applies to all WCA Volunteers and anyone else who is assigned, or given access to, a corporate email. WCA Volunteers using a WCA email represent the World Cube Association and must communicate accordingly.

Reviewer: ```suggestion
The purpose of this policy is to outline the guidelines and requirements of individual user accounts and group accounts on the worldcubeassociation.org domain. This policy applies to all WCA Volunteers and anyone else who is assigned, or given access to, a corporate email account. WCA Volunteers using a WCA email account represent the World Cube Association and must communicate accordingly.
```

## Policy

> All WCA Volunteers must have a valid WCA email address linked to an account on worldcubeassociation.org. This email is assigned by a G Suite administrator and must be claimed and linked before the WCA Volunteer is appointed on the WCA website.

Reviewer: ```suggestion
All WCA Volunteers must have a valid WCA email address linked to an email account on worldcubeassociation.org. This email account is assigned by a Google Workplace administrator and must be claimed and linked before the WCA Volunteer is appointed on the WCA website.
```
G Suite is now Google Workplace

WCA Volunteers should not use their WCA email for personal reasons.

### Email Signature
WCA Volunteers are encouraged to create a professional email signature. The email signatures should not contain logos or unnecessary textual references. Here is a sample acceptable email signature:

**Mike Miller**
*Member of the WCA Results Team*
*WCA Delegate for United States (New York)*

### Disciplinary Action

> WCA Volunteers who do not adhere to this policy may face disciplinary action up to and including dismissal from their Volunteer role.

Reviewer: ```suggestion
WCA Volunteers who do not adhere to this policy may face disciplinary action, including dismissal from their Volunteer role.
```
The previous edited changes the meaning slightly, as it now suggests that disciplinary action beyond such dismissal, e.g. a ban, is now not possible, whereas formerly it was not precluded.

### Account Deletion and Suspension

> - When an individual is no longer a WCA Volunteer, their WCA email shall be suspended.

Reviewer: ```suggestion
- When an individual is no longer a WCA Volunteer, their WCA email account shall be suspended.
```

>    - If an individual becomes a WCA Volunteer again, their WCA email shall be reinstated.

Reviewer: ```suggestion
   - If an individual becomes a WCA Volunteer again, their WCA email account shall be reinstated.
```

> - The WCA G Suite administrator reserves the right to delete WCA emails in certain circumstances.

Reviewer: ```suggestion
- The WCA Google Workplace administrator reserves the right to delete WCA email accounts in certain circumstances.
```

>    - After a period of 7 years after a WCA Volunteer’s access is revoked, their WCA email, and all related account emails, shall be deleted.

Reviewer: ```suggestion
   - Following a period of 7 years after a WCA Volunteer’s access is revoked, their WCA email account, and all related account emails, shall be deleted.
```
After ... after is awkward
